crime-applications-adaptor/src/test/resources/data/criminalapplicationsdatastore/MaatApplication_6000288.json

```diff
@@ -37,5 +38,5 @@
     "hearing_court_name": "Teesside Magistrates' Court",
     "hearing_date": "2023-05-10"
   },
-  "interests_of_justice": []
+  "ioj_bypass": false
```

Comment:
@mtac50 Hi Mike, in case it matters, just wanted to note that I believe in the fixtures the `ioj_bypass` value is flipped 😄 
For instance here, according to the original fixture (it didn't have `interests_of_justice`, array is empty) the `ioj_bypass` would be true, meaning, the IoJ reasons page was skipped.

In other fixtures where originally we had `interests_of_justice` (array not empty) the `ioj_bypass` would be correspondingly `false` (meaning the IoJ page was presented to the provider and they entered stuff).

Not sure if it matters to you as I say but just in case! Ping me if you need any more detail!